prov/sockets/src/sock_mr.c

```diff
@@ -95,20 +96,74 @@ static int sock_mr_bind(struct fid *fid, struct fid *bfid, uint64_t flags)
 	return 0;
 }
 
+static int sock_mr_get_raw(struct sock_mr *mr, struct fi_mr_raw_attr *attr) {
+	size_t copy_size;
+	void *copy_src;
+	int ret;
+
+	if (!attr) return -FI_EINVAL;
+	if (!attr->base_addr) return -FI_EINVAL;
+	if (!attr->raw_key) return -FI_EINVAL;
+	if (!attr->key_size) return -FI_EINVAL;
```

Comment:
please follow linux kernel coding guidelines.  The above if statements can be merged into a single check